<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-IRE7K3E" failures="1" tests="2" name="Buy the Stocks" time="38.238" errors="0" timestamp="2022-06-03T12:15:55 IST">
  <testcase classname="TestCases.DoLogin" name="doLogin" time="27.344">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	My mistake">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
My mistake
at Utilty.GenericKeyword.assertAll(GenericKeyword.java:179)
at TestCases.DoLogin.doLogin(DoLogin.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at Runner.TestNGRunner.run(TestNGRunner.java:80)
at Runner.Runner.main(Runner.java:31)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
  <testcase classname="TestCases.DoLogin" name="selectPortfolio" time="8.584"/>
</testsuite> <!-- Buy the Stocks -->
